package.json

```diff
@@ -161,7 +161,8 @@
             "nim": "nim compile --verbosity:0 --hints:off --run",
             "lisp": "sbcl --script",
             "kit": "kitc --run",
-            "v": "v run"
+            "v": "v run",
+            "scss": "scss --style expanded"
```

Comment:
Is it scss or sass? https://sass-lang.com/install
Sorry, I am not quite familiar with this. Just want to have a double confirm. :)

Reply:
Both exist :)

```bash
sorokin@here:~ $ sass -v
Sass 3.4.21 (Selective Steve)
sorokin@here:~ $ scss -v
Sass 3.4.21 (Selective Steve)
```

Reply:
Thanks for double confirm. :)
BTW, is `"scss": "scss"` enough? It seems that `--style expanded` is already the default parameter.

Reply:
Maybe it depends on which scss compiler you have installed.

`scss`
```css
.test {
  foo: bar; }
```

`scss --style expanded`
```css
.test {
  foo: bar;
}
```

In any case, I'd prefer to be explicit, we're not typing in the full command manually so there's no penalty for writing it out even if it was the default.

Reply:
Re: scss vs sass
https://stackoverflow.com/a/22069780/6398754
https://stackoverflow.com/a/31403160/6398754

Reply:
Awesome! Thanks for the detailed explanation!